docs/user-guide/advanced_topics/monte_carlo.md

```diff
@@ -0,0 +1,23 @@
+# Monte Carlo Simulations
```

Comment:
The entire text for this section seems to be in the advanced examples, and it feels more application/example relevant than other topics in the user guide.

Instead of duplicating the content, can we remove this file (which doesn't appear to be linked from anywhere either)?

Reply:
@allisonmcampbell, can you figure out if we still need this file? If not, feel free to pull it out.